manifests/config.pp

```diff
@@ -0,0 +1,45 @@
+# Configuration items for GDM
+#
+#
+class gdm::config {
+  assert_private()
+
+  gnome::dconf::profile { 'gdm':
```

Comment:
Seems like dconf should be in its own module.  Otherwise gnome and gdm are inappropriately coupled

Reply:
Ticket created at https://simp-project.atlassian.net/browse/SIMP-4935

This is more involved than just a copy/paste due to the need for a native type.